app/controllers/grades_controller.rb

```diff
@@ -88,6 +88,27 @@ def view_my_scores
     @summary = sum.summary
     @avg_scores_by_round = sum.avg_scores_by_round
     @avg_scores_by_criterion = sum.avg_scores_by_criterion
+
+
+
+
+    @review_mappings = SelfReviewResponseMap.where(reviewer_id: params[:id])
+    @sorted_responses = Array.new
+    @prev = Response.where(:map_id => @review_mappings[0].id)
```

Comment:
If line 95 returns nothing, then no self review exists. If no self review exists then this page wont be invoked